frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -1,35 +1,37 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useActions, useValues } from 'kea'
 import dayjs from 'dayjs'
 import { parsePeopleParams, trendsLogic } from 'scenes/trends/trendsLogic'
 import { DownloadOutlined } from '@ant-design/icons'
-import { Modal, Button, Spin } from 'antd'
+import { Modal, Button, Spin, Input } from 'antd'
 import { PersonsTable } from 'scenes/persons/PersonsTable'
-import { Link } from 'lib/components/Link'
-import { ArrowRightOutlined, ClockCircleOutlined } from '@ant-design/icons'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 import { ViewType } from 'scenes/insights/insightLogic'
-
+import { FEATURE_FLAGS } from 'lib/constants'
 interface Props {
     visible: boolean
     view: ViewType
     onSaveCohort: () => void
 }
 
 export function PersonModal({ visible, view, onSaveCohort }: Props): JSX.Element {
-    const { people, filters, peopleModalURL, loadingMorePeople } = useValues(
+    const { people, filters, loadingMorePeople, firstLoadedPeople } = useValues(
         trendsLogic({ dashboardItemId: null, view })
     )
-    const { setShowingPeople, loadMorePeople } = useActions(trendsLogic({ dashboardItemId: null, view }))
+    const { setPersonsModalFilters } = useActions(trendsLogic({ dashboardItemId: null, view }))
+    const { setShowingPeople, loadMorePeople, setPeople } = useActions(trendsLogic({ dashboardItemId: null, view }))
     const { featureFlags } = useValues(featureFlagLogic)
-
+    const [searchTerm, setSearchTerm] = useState('')
     const title =
         filters.shown_as === 'Stickiness'
             ? `"${people?.label}" stickiness ${people?.day} day${people?.day === 1 ? '' : 's'}`
             : filters.display === 'ActionsBarValue' || filters.display === 'ActionsPie'
             ? `"${people?.label}"`
             : `"${people?.label}" on ${people?.day ? dayjs(people.day).format('ll') : '...'}`
-    const closeModal = (): void => setShowingPeople(false)
+    const closeModal = (): void => {
+        setShowingPeople(false)
+    }
+
```

Comment:
Technically we need to provide return types to only `export`ed functions. I guess this wasn't initially your code, but just though I'd share.

Reply:
I tried to remove this but eslint yells at me if I don't add a return type here 😱